Runtime/Tracking/Velocity/XRNodeVelocityEstimator.cs

```diff
@@ -27,13 +35,16 @@ public class XRNodeVelocityEstimator : VelocityTracker
         protected override Vector3 DoGetVelocity()
         {
             GetNodeState().TryGetVelocity(out Vector3 result);
+            result = RelativeTo.transform.rotation * result;
```

Comment:
This should be wrapped in a null check because `RelativeTo` is optional:

```
result = RelativeTo != null ? RelativeTo.transform.rotation * result : result;
```